modules/afsocket/afsocket-parser.c

```diff
@@ -66,6 +66,7 @@ static CfgLexerKeyword afsocket_keywords[] =
   { "sni",                KW_SNI },
   { "allow_compress",     KW_ALLOW_COMPRESS },
   { "ocsp_stapling_verify", KW_OCSP_STAPLING_VERIFY },
+  { "conf_cmds",          KW_CONF_CMDS},
```

Comment:
Can we call this feature `openssl-conf-cmd()`?
It would clarify the possible future direction when we introduce an alternative TLS library, which does not support OpenSSL commands.